script.js

```diff
@@ -0,0 +1,79 @@
+const addButton1 = document.querySelector(".button-add-1");
+const addButton2 = document.querySelector(".button-add-2");
+const addButton3 = document.querySelector(".button-add-3");
+
+addButton1.addEventListener("click", addToList1);
+document.querySelector(".text-1").addEventListener("keydown", function (e) {
+    if (e.key === "Enter") {
+      addToList1();
+    }
+});
+addButton2.addEventListener("click", addToList2);
+document.querySelector(".text-2").addEventListener("keydown", function (e) {
+    if (e.key === "Enter") {
+      addToList2();
+    }
+});
+addButton3.addEventListener("click", addToList3);
+document.querySelector(".text-3").addEventListener("keydown", function (e) {
+    if (e.key === "Enter") {
+      addToList3();
+    }
+});
+
+//I really tried to get this to work as a function with arguments so there could be only one function instead of three but there is some horrible arcane
+//secret involved in getting "click" to work with a function that has parameters FOR SOME REASON??????????? so... this is what we get.
+function addToList1(){
+    let textBox = document.querySelector(".text-1");
+    if (textBox.value === ""){
+        document.querySelector(".text-1").focus();
```

Comment:
You set the textbox above, so you can use that here instead

```textBox.focus()```

Reply:
Fixed with [ace7db8](https://github.com/Vhynn/ToDoApp/pull/7/commits/ace7db87b0abe0dd1aa16ec2df25c088cea67d93).